src/src/pages/capabilities/NewCapabilityDialog.js

```diff
@@ -39,6 +69,18 @@ export default function NewCapabilityDialog({
     setFormData((prev) => ({ ...prev, ...{ description: newValue } }));
   };
 
+  async function changeInvitation(e) {
+    e.preventDefault();
+    const regex = /[^,]*$/;
+    setIsUserSearchActive(true);
+    const adValue = e?.target?.value.match(regex)[0];
+    const value = e?.target?.value;
```

Comment:
maybe rename these to reflect what the type of the values are. Value is very generic and from reading the code I don't know the type.

Reply:
Adressed in Addressed in https://github.com/dfds/selfservice-portal/pull/133/commits/cb4b6bfc44c8d7dff3c4ea8e1c66e2dc416c35bd